server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java

```diff
@@ -474,7 +474,8 @@ public static RealmRepresentation toRepresentation(KeycloakSession session, Real
 
         session.clientPolicy().updateRealmRepresentationFromModel(realm, rep);
 
-        rep.setAttributes(stripRealmAttributesIncludedAsFields(realm.getAttributes()));
+        // Append realm attributes to representation
+        rep.getAttributes().putAll(stripRealmAttributesIncludedAsFields(realm.getAttributes()));
```

Comment:
Attributes in present realm representation was overridden by `RealmModel` attributes. These attributes should be appended.